keyboards/zoop/leon/leons/info.json

```
@@ -0,0 +1,112 @@
+{
+    "keyboard_name": "Leon",
+    "manufacturer": "Zookeepers",
+    "url": "",
+    "maintainer": "HuangJuanLR",
+    "usb": {
+        "vid": "0x4C52",
+        "pid": "0x4C80",
+        "device_version": "0.0.2"
+    },
+    "layouts": {
+        "LAYOUT_all": {
+            "layout": [
+                { "label": "Esc", "x": 0, "y": 0 },
+                { "label": "F1", "x": 1.25, "y": 0 },
+                { "label": "F2", "x": 2.25, "y": 0 },
+                { "label": "F3", "x": 3.25, "y": 0 },
+                { "label": "F4", "x": 4.25, "y": 0 },
+                { "label": "F5", "x": 5.5, "y": 0 },
+                { "label": "F6", "x": 6.5, "y": 0 },
+                { "label": "F7", "x": 7.5, "y": 0 },
+                { "label": "F8", "x": 8.5, "y": 0 },
+                { "label": "F9", "x": 9.75, "y": 0 },
+                { "label": "F10", "x": 10.75, "y": 0 },
+                { "label": "F11", "x": 11.75, "y": 0 },
+                { "label": "F12", "x": 12.75, "y": 0 },
+                { "label": "F13", "x": 14, "y": 0 },
+                { "label": "Print Screen", "x": 15.25, "y": 0 },
+                { "label": "Scroll Lock", "x": 16.25, "y": 0 },
+                { "label": "Pause", "x": 17.25, "y": 0 },
+
+                { "label": "~", "x": 0, "y": 1.25 },
+                { "label": "1", "x": 1, "y": 1.25 },
+                { "label": "2", "x": 2, "y": 1.25 },
+                { "label": "3", "x": 3, "y": 1.25 },
+                { "label": "4", "x": 4, "y": 1.25 },
+                { "label": "5", "x": 5, "y": 1.25 },
+                { "label": "6", "x": 6, "y": 1.25 },
+                { "label": "7", "x": 7, "y": 1.25 },
+                { "label": "8", "x": 8, "y": 1.25 },
+                { "label": "9", "x": 9, "y": 1.25 },
+                { "label": "0", "x": 10, "y": 1.25 },
+                { "label": "_", "x": 11, "y": 1.25 },
+                { "label": "+", "x": 12, "y": 1.25 },
+                { "label": "LBkspc", "x": 13, "y": 1.25 },
```

Moving `RBkspc` up from Line 81.

```suggestion
                { "label": "LBkspc", "x": 13, "y": 1.25 },
                { "label": "RBkspc", "x": 14, "y": 1.25 },
```